### DISCUSSION TITLE
How to change generate voice?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [coqui-ai/TTS#599] [General]
Can I generate a male voice or voice with some other accent. If yes then how?

**Comment by Sadam1195** [comment]
> Can I generate a male voice or voice with some other accent. If yes then how?

Yes, use a pretrained model checkpoint and do transfer learning by proving new data and train it with resume_training. @talhaanwarch

**Reply by talhaanwarch** [reply]
Is not their pre built model for male voice?

**Reply by thorstenMueller** [reply]
As @Sadam1195 said.
1.) Find an existing pretrained male voice model for your language
2.) Record a dataset (or find an existing) with the accent you're looking for
3.) Finetune that pretrained model with the new "accent" voice dataset

But you have to find an dataset for your requested accent or record one, if it's your own accent.

If you hoped to change accent by just changing a config value that's not possible.

**Reply by talhaanwarch** [reply]
thanks both of you. may be you get confused with word accent. I want to know is there any default male voice in coqui-ai models no matter what accent is because the current voice hear like a female voice

**Reply by Sadam1195** [reply]
If you are talking about English then to my knowledge there is no male English voice but there is non-binary voice (sam) which you can use which is not female voice. @talhaanwarch

**Reply by talhaanwarch** [reply]
I tried this 
`!tts --text "Hello, we are testing" --model_name "tts_models/en/sam/tacotron-DDC" --out_path output.wav`
but I am getting this error
`RuntimeError:  [!] Use 🐸TTS <= v0.0.13 for this model. Current version does not support phoneme based models.`

**Reply by Sadam1195** [reply]
You need to downgrade your TTS version to 0.0.13 for `sam` model to work. As of new version, phoneme support has been dropped and `sam` model was trained with phonemes inputs. @talhaanwarch

**Reply by talhaanwarch** [reply]
I already tried `!pip install TTS==0.0.13` but i got this error
ERROR: No matching distribution found for TTS==0.0.13

**Reply by Sadam1195** [reply]
Try 0.0.13.1

**Reply by talhaanwarch** [reply]
Now i am getting this error 
`ModuleNotFoundError: No module named 'numba.decorators'`
when I do `!tts --list_models` on v0.0.13.1

**Reply by Sadam1195** [reply]
Coqui just released new version. Can you try that?

**Reply by talhaanwarch** [reply]
i am unable to use sam voice in new version

**Reply by Sadam1195** [reply]
Do you mean you tried version 1.0 as well?

**Reply by talhaanwarch** [reply]
> I tried this
> `!tts --text "Hello, we are testing" --model_name "tts_models/en/sam/tacotron-DDC" --out_path output.wav`
> but I am getting this error
> `RuntimeError: [!] Use 🐸TTS <= v0.0.13 for this model. Current version does not support phoneme based models.`

I have tried the latest version and this is the issue

**Reply by Sadam1195** [reply]
Please try earlier versions. Check releases here https://pypi.org/project/TTS/#history

**Reply by erogol** [reply]
> > I tried this
> > `!tts --text "Hello, we are testing" --model_name "tts_models/en/sam/tacotron-DDC" --out_path output.wav`
> > but I am getting this error
> > `RuntimeError: [!] Use 🐸TTS <= v0.0.13 for this model. Current version does not support phoneme based models.`
> 
> I have tried the latest version and this is the issue

this is not the latest version.  Lastest version has no such warning

**Reply by RobinE89** [reply]
hey. i have the same problem. 
-pip install tts-  ->> throws errors [windows]:
Collecting tts
  Using cached TTS-0.1.1.tar.gz (1.3 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
  Using cached TTS-0.1.0.tar.gz (1.3 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
.......
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\.....\\AppData\\Local\\Temp\\pip-install-uwydskdt\\tts_bd.....\\requirements.txt'

then it automatically installs version 0.0.15

and this leads to the errors described by Talhaanwarch

**Reply by RobinE89** [reply]
seems like "gruut" is the problem itself. 
ERROR: Command errored out with exit status 1:
     command: '......' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'........\\Local\\Temp\\pip-install-5vdygpx7\\gruut_2e0dcecefb60451d873ada7f5e054b0f\\setup.py'"'"'; __file__='"'"'.....\\Local\\Temp\\pip-install-5vdygpx7\\gruut_2e0dcecefb60451d873ada7f5e054b0f\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base .......\Local\Temp\pip-pip-egg-info-s4263635'
         cwd: .......\Local\Temp\pip-install-5vdygpx7\gruut_2e0dcecefb60451d873ada7f5e054b0f\
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "........\Local\Temp\pip-install-5vdygpx7\gruut_2e0dcecefb60451d873ada7f5e054b0f\setup.py", line 15, in <module>
        long_description = readme_path.read_text()
      File ".......\Python36\lib\pathlib.py", line 1197, in read_text
        **return f.read()**
      File "........\Python36\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    **UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 3166: character maps to <undefined>**

**Reply by RobinE89** [reply]
@talhaanwarch take a look here: [ERROR: No matching distribution found for gruut[cs,de,es,fr,it,nl,pt,ru,sv]~=1.2.0 ](https://github.com/coqui-ai/TTS/discussions/632)
the error has been fixed and therefore you should now be able to install the latest TTS version on windows
( tts_models/en/sam/tacotron-DDC ->I just tested it there and here it goes )
